File: Ubuntu/Trigonometric Ratios.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>
using namespace std;

int main ()
{
    double pi = atan(1) * 4;
    string angleType;
    cout << "Degrees (d) or radians (r)?\n";
    cin >> angleType;
    double orig;
    cout << "What is the angle measure? (decimal)\n";
    cin >> orig;
    if (angleType == "d")
    {
        orig = orig * pi/180;
        cout << "sin θ = " << sin(orig) << endl;
        cout << "cos θ = " << cos(orig) << endl;
        cout << "tan θ = " << tan(orig) << endl;
        cout << "csc θ = " << 1/sin(orig) << endl;
        cout << "sec θ = " << 1/cos(orig) << endl;
        cout << "cot θ = " << 1/tan(orig) << endl;
    }
    else
    {
        bool radType;
        cout << "Is it in terms of pi? (Y=1)\n";
        cin >> radType;
        if (radType == true)
        {
            orig = pi * orig;
            cout << "sin θ = " << sin(orig) << endl;
            cout << "cos θ = " << cos(orig) << endl;
            cout << "tan θ = " << tan(orig) << endl;
            cout << "csc θ = " << 1/sin(orig) << endl;
            cout << "sec θ = " << 1/cos(orig) << endl;
            cout << "cot θ = " << 1/tan(orig) << endl;
        }
        else
        {
            cout << "sin θ = " << sin(orig) << endl;
            cout << "cos θ = " << cos(orig) << endl;
            cout << "tan θ = " << tan(orig) << endl;
            cout << "csc θ = " << 1/sin(orig) << endl;
            cout << "sec θ = " << 1/cos(orig) << endl;
            cout << "cot θ = " << 1/tan(orig) << endl;
        }
    }
    return 0;
}

File: Visual Basic .NET/Trigonometric Ratios.vb
Module Module1

    Sub Main()
        Dim pi As Double = 3.14159265
        Console.WriteLine("Degrees (0) or radians (1)?")
        Dim ang As Double = Console.ReadLine()
        Console.WriteLine("What is the angle measure?")
        Dim o As Double = Console.ReadLine()
        If ang = 0 Then
            o = o * pi / 180
        Else
            Console.WriteLine("In terms of pi (1) or not (0)?")
            Dim rad As Double = Console.ReadLine()
            If rad = 1 Then
                o = pi * o
            End If
        End If
        Console.WriteLine("sin = {0}{6}cos = {1}{6}tan = {2}{6}csc = {3}{6}sec = {4}{6}cot = {5}", Math.Sin(o), Math.Cos(o), Math.Tan(o), 1 / Math.Sin(o), 1 / Math.Cos(o), 1 / Math.Tan(o), Environment.NewLine)
    End Sub

End Module